website/static/js/institutionProjectSettings.js

```diff
@@ -2,105 +2,189 @@
 var $ = require('jquery');
 var ko = require('knockout');
 var Raven = require('raven-js');
+var m = require('mithril');
 var bootbox = require('bootbox');
 
 var $osf = require('js/osfHelpers');
+var projectSettingsTreebeardBase = require('js/projectSettingsTreebeardBase');
 
 var ViewModel = function(data) {
     var self = this;
     self.loading = ko.observable(true);
     self.showAdd = ko.observable(false);
     self.institutionHref = ko.observable('');
     self.userInstitutions = window.contextVars.currentUser.institutions;
```

Comment:
This is the line that is causing an error in the karma tests you added. As suggested in my previous comment, we really shouldn't be using `contextVars` in this module, but that refactor is outside the scope of this PR. I've created a new ticket https://openscience.atlassian.net/browse/OSF-6416 . 

For now, you could use `lodash.get` which allows you to do nested attribute access;

```js
    self.userInstitutions = lodashGet('window.contextVars.currentUser.institutions') || [];
```